.github/workflows/metal-run-microbenchmarks.yaml

```diff
@@ -42,6 +42,8 @@ jobs:
           PIPELINE_TYPE="microbenchmarks"
           if [ "${{ matrix.runner-info.ccl }}" == "true" ]; then
             PIPELINE_TYPE="ccl_microbenchmarks"
+          else
+            TT_METAL_SLOW_DISPATCH_MODE=1 ./tests/scripts/run_tunneler_tests.sh --machine-type ${{ matrix.runner-info.runs-on[0] }}
```

Comment:
Ooohh that's smart but a dangerous assumption...  we should be careful to run all configs of this to be sure we don't lose track of `[0] == "N300"` in this case